tests/005-test-devmapper-cleanup

```diff
@@ -0,0 +1,61 @@
+source $SRCDIR/libtest.sh
+
+cleanup_local() {
+  local test_status
+  local vg_name=$1
+  local devs=$2
+  $DSSBIN --reset >> $LOGS 2>&1
+  test_status=0
```

Comment:
Can we capture return code of DSSBIN and make sure it is 0 other test failed anyway.